### PR TITLE
Zakonypreludi

### PR DESCRIPTION
**Pull request by MilanKral**

### CODE REVIEW

File: src/chrome/content/rules/Zakonypreludi.sk.xml [hunk 1]
```diff
@@ -0,0 +1,21 @@
+<!--
+	Nonfunctional hosts in *.zakonypreludi.sk:
+
+	certificate mismatch:
+	    - zakonypreludi.sk
+	    - mobile.zakonypreludi.sk
```
Comment (J0WI) [inline src/chrome/content/rules/Zakonypreludi.sk.xml]: `m.zakonypreludi.sk`
Wildcard DNS

Comment (J0WI) [reply src/chrome/content/rules/Zakonypreludi.sk.xml]: Please also add a hint about the wildcard DNS.

**Comment by jeremyn** [comment]
@MilanKral Are you still interested in working on this?

**Comment by J0WI** [comment]
@MilanKral could you please add a hint about the wildcard DNS record?

**Pull request merged**